app/controllers/expenses_controller.rb

```diff
@@ -0,0 +1,73 @@
+class ExpensesController < ApplicationController
+  before_action :set_expense, only: %i[show edit update destroy]
+
+  # GET /expenses or /expenses.json
+  def index
+    @expenses = Expense.all
+  end
+
+  # GET /expenses/1 or /expenses/1.json
+  def show; end
+
+  # GET /expenses/new
+  def new
+    @expense = Expense.new
+    @groups = Group.all
+  end
+
+  # GET /expenses/1/edit
+  def edit; end
+
+  # POST /expenses or /expenses.json
+  def create
```

Comment:
- [ ] Good job so far 👏 . When I tried to create new expenses(transactions)  as you can see in the screenshot below ⬇️  I think you could check the relationship between the author and the expense(transactions) 😃  While fixing the issue please make sure the transactions follow the requirement below ⬇️ 

**1. For a given category, the list of transactions is presented, ordered by the most recent.
2. At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).
3. There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
4. When the user clicks on the "Back" button (<), the user navigates to the home page.**


"Add a new transaction" page

**1. The user fills out a form to create a new transaction with:

-    name (mandatory)
-    amount (mandatory)
-    categories (mandatory at least one)

2. The user click (or taps) the "Save" button to create the new transaction, and is taken to the transactions page for that category.
3. When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category.**

#### You page with errors
![unnamed](https://github.com/Chaseknis/budget_app/assets/6615394/a4102cb3-0645-41c6-a815-629e59edeed2)